alpenhorn/client/__init__.py

```diff
@@ -21,7 +21,6 @@
 def cli():
     """Client interface for alpenhorn. Use to request transfers, mount drives,
     check status etc."""
-    pass
```

Comment:
Can you have empty method body like this? I always thought "pass" was required here.

Reply:
Yeah, I think if you have a doc string it's fine as it counts as part of a function body.